### PR TITLE
Updates artwork filter fonts per QA feedback

### PR DESCRIPTION
**Pull request by ashleyjelks**
Updates font styling based on feedback from the QA of https://artsyproduct.atlassian.net/browse/FX-1781.

- Can we use consistent casing? – Recently Updated and Added use Title Casing and Artwork year uses sentence casing. Can we use sentence casing?
- “Filter” should be 16px Unica Medium 
- “Clear all” should be 14 px Unica Regular
- “Sort” by and “Default” should be 16px Garamond

### CODE REVIEW

**Comment by ds300** [comment]
:ceiling_cat: hey folks. Recently there has been some difficulty around our use of Garamond in the app and, to summarize, it would be really great if we can stop using it wherever branding is not an issue. Please reach out to Sam C for more details.

**Comment by ashleyjelks** [comment]
> :ceiling_cat: hey folks. Recently there has been some difficulty around our use of Garamond in the app and, to summarize, it would be really great if we can stop using it wherever branding is not an issue. Please reach out to Sam C for more details.

Is the use of Garamond causing this Travis CI failure? 

Going to post a message to FX's designer and Sam C about this issue now.

**Comment by ds300** [comment]
> Is the use of Garamond causing this Travis CI failure?

Nope, this `TypeError: TaskQueue: Error with task : Cannot read property 'body' of null` error is flake and you can just hit the rebuild button and it should go away. We saw it before in the emission repo but it seems to have become more frequent after merging with eigen 😞

File: src/lib/Components/FilterModal.tsx [hunk 1]
```diff
@@ -130,10 +130,10 @@ export class FilterOptions extends React.Component<FilterOptionsProps, FilterOpt
               <CloseIcon fill="black100" />
             </CloseIconContainer>
           </Flex>
-          <FilterHeader weight="medium" size="4">
+          <FilterHeader weight="medium" size="4" color="black100">
```
Comment (xtina-starr) [inline src/lib/Components/FilterModal.tsx]: Isn't FilterHeader suppose to be changed to 16px Unica Medium? I'm not super sure where that's happening. Can you point me to that change?

**Comment by xtina-starr** [comment]
@ashleyjelks this looks good. I had one question.

File: src/lib/Components/FilterModal.tsx (hunk 1 above)
Comment (ashleyjelks) [reply src/lib/Components/FilterModal.tsx]: I pulled these sizes from the Zeplin plugin that converts the spec to the Palette component.  The plugin is crashing for me now but I can post a screenshot later.

Comment (xtina-starr) [reply src/lib/Components/FilterModal.tsx]: I also so the definition for `FilterHeader` further down in the file. So all good!

**Pull request merged**